### ISSUE TITLE
Go to previous/ next error, warning, problem does not read the content to the screen reader

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#17099]
1. turn on a screen reader
2. run `Go to previous/next error, warning, problem`
3. expectation: the error, warning, or problem content will be read to me
4. 🐛 nothing is read

Discovered while trying to get `aria-live` to work for presenting the inline ghost text to a screen reader. It appears that `aria-live` does not work even when it's set to `assertive`. Focusing the tooltip reads it, so maybe that's what we should do here

https://github-com/microsoft/vscode/blob/1028dd25f857c6b735a1870bb63653845786a5d5/src/vs/editor/contrib/gotoError/browser/gotoErrorWidget.ts-L62-L63
